Payments/Operation/01_ManageAuthorizations.md

```diff
@@ -0,0 +1,126 @@
+
+After you have checked an authorization transaction that could not be processed by the workflow, you can capture the transaction manually, so that the order process can be continued.
+
+#### Prerequisites
+
+- The status of an authorization transaction is **not** "Success". In this case, further processing has already been continued.<!---Stimmt das oder gibt es noch mehr?-->   
+
+
+#### Procedure
+*Payments > Authorizations > Tab LIST*
+
+![Capture authorization transaction](../../Assets/Screenshots/Payments/Authorizations/ChangeAuthorization.png "[Capture authorization transaction]")   
+
+1. Check the authorization transaction you want to capture, see [Check authorization transaction](./01_ManageAuthorization.md#check-authorization-transaction).
+2. Select the authorization transaction you want to capture by clicking the checkbox on the left.   
+    The editing toolbar is displayed.
+3. Click the [Capture] button.    
+   A confirmation message is displayed. The status of the authorization transaction has changed to "Success".   
+   You can now follow up this case under the menu entry *Payments and captures*. <!---Stefan ist das richtig?--> 
+   
+  
+
+## Cancel authorization transaction
+
+After you have checked an authorization transaction that could not be processed by the workflow, you can cancel the transaction manually, so that the order is cancelled both at Actindo and at the payment service provider. Actindo will then automatically inform the payment service provider about the cancellation. The customer will get his or her money back.
+
+#### Prerequisites
+- The status of an authorization transaction is "Failure" or "Error". 
+- The authorization transaction has not been captured before.
+
+#### Procedure
+
+*Payments > Authorizations > Tab LIST*
+
+![Cancel authorization transaction](../../Assets/Screenshots/Payments/Authorizations/ChangeAuthorization.png "[Cancel authorization transaction]")
+
+1. Identify the communication issue that led to the error. To do this, check the authorization transaction you want to cancel, see [Check authorization transaction](01_ManageAuthorization.md#check-authorization-transaction).
+2. Select the authorization transaction you want to cancel by clicking the checkbox on the left.   
+    The editing toolbar is displayed.
+3. Click the [Cancel] button.   
+   A confirmation message is displayed. 
+   The payment service provider gets the information about the cancellation of the authorization transaction. The payment of the customer, blocked before by the payment service provider, is freed, and the customer get his or her money back.
+   
+
+
+
+## Void authorization transaction
+
+You can void an authorization transaction, if you want to disable the transaction for the Actindo database. For example, for whatever reason the same transaction has been posted twice. In this case, you can void one of these transactions and continue the order process with the other transaction.
```

Comment:
Generell zu "disable the/a transaction": Ich finde "disable" schwer hier, da wir es in Verbindung mit Connection verwenden und hat eine ganz andere Bedeutung. Wie wäre es mit "invalidate/validate", wenn wir über transactions sprechen? (Du hast "valid" schon in diesem Bezug verwendet).

Reply:
done